src/screens/NationDetailsContainer/index.js

```diff
@@ -36,25 +107,31 @@ class NationDetailsContainer extends Component {
     if (_.isEmpty(this.props.wallets)) {
       this._showCreatePrivateKeyAlert();
     } else {
```

Comment:
This else is redundant as far as I can see. 
You can have just: 
```
if (_.isEmpty(this.props.wallets)) {
   this._showCreatePrivateKeyAlert();
}

fn()
```

Reply:
We should call function only if user has wallet, so we need either else or return inside if

Reply:
Ah yeah of course. But go for a return. It's less code.